plugins/org.python.pydev/tests_analysis/com/python/pydev/analysis/OccurrencesAnalyzer2Test.java

```diff
@@ -846,4 +846,27 @@ public void testInvalidEncoding() throws Exception {
     //        );
     //        checkError(1);
     //    }
+
+    public void testNoSelfInZopeInterface() throws Exception {
+        doc = new Document(""
+                + "class I(Interface):\n"
```

Comment:
I think that you should shouldn't really do the test without the actual import:

```
from zope.interface import Interface
```

the problem is that there may be other libraries that have `Interface` declared and you don't want to skip the `self` check on those... something such as:

```
from zope.interface import Interface as I

class Impl(I):
    def method_a():
        pass
```

must also work.

Please make a separate test for the case above too...